src/Components/Molecules/Alert/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect, Dispatch } from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+
+import style from "./Alert.module.scss";
+
+type Props = {
+  showAlert: boolean;
+  setShowAlert: React.Dispatch<React.SetStateAction<boolean>>
```

Comment:
Tem um padrão q a gente bota q eh
```suggestion
  isAlertOpen: boolean;
  setIsAlertOpen: React.Dispatch<React.SetStateAction<boolean>>
```